src/python/m5/ext/pystats/__init__.py

```diff
@@ -30,6 +30,7 @@
 from .statistic import Statistic
 from .storagetype import StorageType
 from .timeconversion import TimeConversion
+from .jsonloader import JSONLoader
```

Comment:
It's not a big problem but I do format this as "Json" for other classes. E.g., "JsonSerializable"

Reply:
resolved [here](https://github.com/darchr/gem5/compare/d974742952a4acfd9a5711f35d25cd42e84d0e70..bf98acaa9d8631db3a97f15eb4bd06524d690d68)